frame/im-online/src/lib.rs

```diff
@@ -608,7 +608,12 @@ impl<T: Trait> pallet_session::OneSessionHandler<T::AccountId> for Module<T> {
 
 			let validator_set_count = keys.len() as u32;
 			let offence = UnresponsivenessOffence { session_index, validator_set_count, offenders };
-			T::ReportUnresponsiveness::report_offence(vec![], offence);
+			if let Err(e) = T::ReportUnresponsiveness::report_offence(vec![], offence) {
+				debug::info!(
```

Comment:
The problem with this, it will never be printed (in wasm). This functionality is only for offchain workers (it requires that the `RuntimeLogger` is intialized). Better to just use `sp_runtime::print`.

Reply:
Yes, but the rest of the code in this modules uses this format so I decided to keep it the same.

Reply:
The code you are referring too is offchain, that is the reason it uses this format.

Reply:
done